backend/etcd/etcd.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// StateDir is the base path for Sensu's local storage.
-	StateDir = "/var/lib/sensu"
+	StateDir = "/var/lib/sensu/sensu-backend"
```

Comment:
I doubt this PR was ever tested because this line doesn't actually have any effect and should be removed. 

It is only referred in the [`NewConfig` function](https://github.com/sensu/sensu-go/blob/321ff41fe8170b37ae4673cce1df2cf29733f702/backend/etcd/etcd.go#L63), but it gets [immediately replaced](https://github.com/sensu/sensu-go/blob/e089a61c0c710dac81f81a9fbcee5beadf4ead5d/backend/backend.go#L68) with whatever config was provided in the configuration. The [default value](https://github.com/sensu/sensu-go/blob/321ff41fe8170b37ae4673cce1df2cf29733f702/backend/cmd/start.go#L268) is indeed `/var/lib/sensu`, which is provided by the [path util](https://github.com/sensu/sensu-go/blob/321ff41fe8170b37ae4673cce1df2cf29733f702/util/path/path.go#L53).

Reply:
Good catch!